cmd/put-main.go

```diff
@@ -0,0 +1,153 @@
+// Copyright (c) 2015-2024 MinIO, Inc.
+//
+// This file is part of MinIO Object Storage stack
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+package cmd
+
+import (
+	"context"
+	"os"
+	"strconv"
+
+	"github.com/dustin/go-humanize"
+	"github.com/minio/cli"
+	"github.com/minio/mc/pkg/probe"
+)
+
+// put command flags.
+var (
+	putFlags = []cli.Flag{
+		cli.IntFlag{
+			Name:  "parallel, P",
+			Usage: "upload number of parts in parallel",
+		},
+		cli.StringFlag{
+			Name:  "part-size, s",
+			Usage: "each part size",
+		},
```

Comment:
```suggestion
		cli.StringFlag{
			Name:  "part-size, s",
			Usage: "each part size",
			Value: "16MiB",
		},
```

Must have a default value.